### PR TITLE
feat:updated schema url

### PR DESCRIPTION
**Pull request by cmcWebCode40**
• I updated the schema URL to use https://common.schemas.verida.io/social/contact/v0.1.0/schema.json

#54

### CODE REVIEW

File: src/plugins/markdown.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import hljs from "highlight.js";
 
 import "highlight.js/styles/atom-one-dark.css";
 
-marked.setOptions({
+marked.default.setOptions({
```
Comment (nick-verida) [inline src/plugins/markdown.js:6]: This doesn't seem to relate to the update for the schema, and broke the build. This showed up in the console.

Not opposed to getting things that need to be fixed into another PR, but they need to be tested very carefully.

```
Uncaught TypeError: marked__WEBPACK_IMPORTED_MODULE_0___default.a.default is undefined
    <anonymous> markdown.js:6
    js app.js:1587
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> index.js:1
    js app.js:1575
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> main.ts:19
    ts app.js:1551
```

**Review by nick-verida — CHANGES_REQUESTED** [review]
Build break. 

Run `npm run serve`, open webpage, blank screen shows. 

View console and this message is there:

```Uncaught TypeError: marked__WEBPACK_IMPORTED_MODULE_0___default.a.default is undefined
    <anonymous> markdown.js:6
    js app.js:1587
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> index.js:1
    js app.js:1575
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> main.ts:19
    ts app.js:1551
```

I've tried deleting `node_modules` and then `npm install`

File: src/plugins/markdown.js (hunk 1 above)
Comment (cmcWebCode40) [reply src/plugins/markdown.js:6]: sorry, I will check that issue, I notice there was an update in the markdown package

**Comment by nick-verida** [comment]
Same error:

```
Uncaught TypeError: marked__WEBPACK_IMPORTED_MODULE_0___default.a.default is undefined
    <anonymous> markdown.js:6
    js app.js:1587
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> index.js:1
    js app.js:1575
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> main.ts:19
    ts app.js:1551
    __webpack_require__ app.js:854
    fn app.js:151
    1 app.js:1696
    __webpack_require__ app.js:854
    checkDeferredModules app.js:46
    <anonymous> app.js:994
    <anonymous> app.js:997
```

It doesn't look like anything was done to address this from the previous review.

**Review by nick-verida — APPROVED** [review]
Seems fixed now!

Merging.
